Documentation/AudioFileFormats.md

```diff
@@ -0,0 +1,93 @@
+
+<img width="822" height="173" alt="image" src="https://github.com/user-attachments/assets/da9adf57-a8da-4962-a6e2-94d86883ba87" />
+
+## Settings Summary
+### Audio quality to request from Audible
+Audiobooks can be requested from Audible as "Normal" quality or "High" quality, matching the settings in the Audible mobile apps. This setting affects the audio bitrate and, sometimes, the number of audio channels. This setting has no effect on the _audio codec_.
+
+### Use Widevine DRM
+This setting will request Audio files protected by Google's Widevine Digital Rights Managements scheme. The practical effects of this option are:
+
+1. When disabled, all audiobooks will be downloaded in the [AAC-LC codec](#aac-lc).
+2. When enabled, (nearly) all non-spatial audiobooks will be downloaded in the [xHE-AAC codec](#xhe-aac). 
+3. This option must be enabled to enable "Request Spatial Audio"
+
+### Request Spatial Audio
+Enable this setting to request audiobooks in a "spatial" ([Dolby Atmos](#dolby-atmos)) audio format. If an audiobook is not available in a spatial format, it will instead be downloaded in the [xHE-AAC codec](#xhe-aac).
+
+### Spatial audio codec
+Choose whether spatial audiobooks are downloaded in the [E-AC-3](#e-ac-3) or [AC-4](#ac-4) format.
+
+### Download my books in the original audio format (Lossless)
+If selected, Audiobooks will be downloaded and saved in the format delivered by audible (which depends on the settings explained above). Libation will not change the audio.
+
+### Download my books as .MP3 files (transcode if necessary).
+If selected, Libation will decode [AAC-LC](#aac-lc), [xHE-AAC](#xhe-aac), and [E-AC-3](#e-ac-3) audiobooks and re-encode them as MP3s using the MP3 encoder settings ([read about LAME MP3 encoder settings](https://lame.sourceforge.io/lame_ui_example.php)). Note that Libation cannot convert [AC-4](#ac-4) audio to MP3.
+
+# Audio Formats
+
+## Traditional Mono and Stereo Formats
+
+### AAC-LC
+#### _Full Name_
+Advanced Audio Coding - Low Complexity
+#### _Description_
+This is the base profile for AAC audio and has existed since AAC's initial release in 1997. It enjoys wide support on nearly every conceivable platform capable of playing digital audio, as ubiquitous as MP3. 
+
+### MP3
+#### _Full Name_
+MPEG-1 Audio Layer III or MPEG-2 Audio Layer III
+#### _Description_
+An older (released in 1991) but still nearly universally supported audio codec. Its audio quality is generally worse than AAC-LC at similar bitrates. Audible delivers some podcasts in MP3 format, but no audiobooks are natively availble as MP3. Libation does support converting Audiobooks delivered in other formats to MP3. Libation can convert [AAC-LC](#aac-lc), [xHE-AAC](#xhe-aac), and [E-AC-3](#e-ac-3) sources to MP3. Note that MP3 format supports a maximum of two audio channels, so E-AC-3 audio will be downsampled to stereo or mono (depending on the Libation's settings).
```

Comment:
1. Maybe delete: "Libation does support converting Audiobooks delivered in other formats to MP3." Seems redundant with what follows. It's also unusual to say "does support" when "supports" is shorter (it almost looks like you intended to say "does not support" but accidentally left out the word "not").
2. On the other hand: maybe keep "libation supports", but then just have the next sentence say the one format we CANNOT convert, rather than giving a long list of formats but not mentioning the one we can't hande?